lib/LaTeXML/Post/MathML.pm

```diff
@@ -984,26 +984,23 @@ sub pmml_script_mid_layout {
       $base = pmml($base); }
     return $base; }
   else {
-    if (scalar(@$midscripts) > 1) {
-      Error("unexpected", 'multiple-mid-level-scripts', undef,
-        "Multiple mid-level (limit) scripts; extras are DROPPED!", "base: " . $base->toString(),
-        map { @$_ } @$midscripts); }
-    { local $LaTeXML::MathML::NOMOVABLELIMITS = 1;
-      ##### TRY this to block an extra mstyle
-      local $LaTeXML::MathML::STYLE = $base->getAttribute('mathstyle') || $LaTeXML::MathML::STYLE;
-      $base = pmml($base); }
+    local $LaTeXML::MathML::NOMOVABLELIMITS = 1;
+    ##### TRY this to block an extra mstyle
+    local $LaTeXML::MathML::STYLE = $base->getAttribute('mathstyle') || $LaTeXML::MathML::STYLE;
     # Get the (possibly padded) over & under scripts (if any)
```

Comment:
You don't really want these 2 locals scoped to the entire routine, but just to formatting the base.